app/locale/en.coffee

```diff
@@ -137,6 +137,7 @@
     poll: "Poll"  # Tooltip on poll button from /play
     next: "Next"  # Go from choose hero to choose inventory before playing a level
     change_hero: "Change Hero"  # Go back from choose inventory to choose hero
+    change_hero_language: "Change Hero or Language"
```

Comment:
Made a new field because "Change Hero or Language" doesn't exist and `change_hero` is used in two other places.